metacat-thrift/src/main/java/com/netflix/metacat/thrift/CatalogThriftHiveMetastore.java

```diff
@@ -841,7 +841,7 @@ public CurrentNotificationEventId get_current_notificationEventId() throws TExce
     public Database get_database(final String name) throws TException {
         return requestWrapper("get_database", new Object[]{name}, () -> {
             final String databaseName = normalizeIdentifier(name);
-            final DatabaseDto dto = v1.getDatabase(catalogName, databaseName, true);
+            final DatabaseDto dto = v1.getDatabase(catalogName, databaseName, true, false);
```

Comment:
We stopped return the table names from get_database() call. Will this break anything?

Reply:
In this method, the list of table names is not used.